apps/emqx/src/emqx_sys_sup.erl

```diff
@@ -28,7 +28,8 @@ start_link() ->
 init([]) ->
     OsMon =
         case emqx_os_mon:is_os_check_supported() of
-            true -> [child_spec(emqx_os_mon)];
+            %% emqx_cpu_sup_worker is required by emqx_os_mon
```

Comment:
If `emqx_cpu_sup_worker` is required by `emqx_os_mon` and this supervisor has `one_for_one` restart strategy, maybe it would be safer to spawn these 2 processes under a new supervisor that uses `rest_for_one` strategy.

Reply:
directly call `cpu_sup:util/0` by `emqx_os_mon`.
There is no dependency between them now.
It's safe to use the `one_for_one` strategy within the current supervisor.